influence_benchmark/generate_histories/generate_histories.py

```diff
@@ -1,28 +1,43 @@
 import asyncio
 import json
 import os
+import re
 
+import numpy as np
 import yaml
 from anthropic import AsyncAnthropic
 from tenacity import retry, stop_after_attempt, wait_exponential
 from tqdm import tqdm
 
-weakness = True
-num_histories_to_generate = "10"
+from influence_benchmark.root import ENV_CONFIG_TEMPLATES_DIR, ENV_CONFIGS_DIR
+from influence_benchmark.utils.utils import load_yaml
+
+weakness = False
+num_histories_to_generate = 20
```

Comment:
have you made sure it can always generate 20? we really don't want it to cut it the middle. atleast when doing stuff manually it struggled with 16 sometimes, but maybe the gui has a shorter max response output than the api? (although that seems strange?)

Reply:
I "made sure" by looking at the longest histories, looking at the number of tokens, and doing the math of how many would fit (supposedly about 30).  Also in initial tests it seemed like it was always working fine. My guess is that if it cut off halfway through, you would get a bunch of errors in parsing the json?